### PR TITLE
Add LocalCluster.updateCatalog method

### PR DESCRIPTION
**Pull request by n3world**

### CODE REVIEW

File: tests/frontend/org/voltdb/regressionsuites/LocalCluster.java [hunk 1]
```diff
@@ -525,13 +525,12 @@ public boolean compile(VoltProjectBuilder builder, final String voltRootPath) {
...
-            m_initialCatalog = builder.compile(templateCmdLine.jarFileName(), m_siteCount, m_hostCount, m_kfactor, null, m_clusterId);
-            m_compiled = m_initialCatalog != null;
-            templateCmdLine.pathToDeployment(builder.getPathToDeployment());
-            m_voltdbroot = builder.getPathToVoltRoot().getAbsolutePath();
-        }
-        return m_compiled;
+        return compile(builder, null);
+    }
+
+    public boolean forceCompile(VoltProjectBuilder builder) {
```
Comment (manjujames) [inline tests/frontend/org/voltdb/regressionsuites/LocalCluster.java]: Why do we need force compile?

Comment (n3world) [reply tests/frontend/org/voltdb/regressionsuites/LocalCluster.java]: I needed to compile a new configuration so I could enable dr for the test for https://github.com/VoltDB/pro/pull/2490

Comment (manjujames) [reply tests/frontend/org/voltdb/regressionsuites/LocalCluster.java]: Oops, I only saw the first commit!

Comment (manjujames) [reply tests/frontend/org/voltdb/regressionsuites/LocalCluster.java]: BTW, you can dynamically update deployment file. I think there are other tests that do that.

**Pull request merged**